package.json

```diff
@@ -26,7 +26,10 @@
     "@angular/router": "16.1.7",
     "@ngneat/transloco": "^5.0.2",
     "@nguniversal/express-engine": "^16.1.1",
+    "@ngx-translate/core": "^15.0.0",
+    "@ngx-translate/http-loader": "^8.0.0",
     "@nrwl/angular": "16.6.0",
+    "bootstrap-icons": "^1.10.5",
```

Comment:
We use font-awesome icons. I'll add a CDN for this that you can get by syncing the latest changes